Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -197,10 +201,14 @@ private Vector3 InitialPosition
 
         #endregion
 
+        private void Awake()
+        {
+            initialOffsetMovingVisuals = PushSpaceSourceTransform.localPosition;
```

Comment:
>PushSpaceSourceTransform.localPosition [](start = 41, length = 38)

This transform may come from another game object that has not awaken yet and that may modify its transform on Awake(). To make sure there are no initialization order issues, I think it is better to do this in Start(). There is a good write up about Awake vs Start here: [https://www.tangledrealitystudios.com/development-tips/awake-vs-start-vs-onenable-and-when-to-use-them/](https://www.tangledrealitystudios.com/development-tips/awake-vs-start-vs-onenable-and-when-to-use-them/). The general rule is: things that rely on other game objects are better done in Start().